### DISCUSSION TITLE
Order Email Confirmation / Wishlist / Compare / Contact Form

### DISCUSSION THREAD

**Discussion by AlohaJeff** [breezefront/community#18] [1. Help]
We're running up against a number of unforeseen issues with Breeze Evolution.

**Email Templates:**
It appears that Breeze Evolution is making changes to our previously stylized Email Templates. The style of the order confirmation email is very different than what we are used to seeing. I thought that perhaps when the "install" was run it changed the settings. However when I check the configuration/sales emails the correct template appears to be selected. Why and How is Breeze Evolution overriding the templates? Can we simply rename the vendor/swissup/theme-frontend-breeze-blank/web/css/email folder to prevent any "theming" that Breeze is trying to do?

**Wishlist**
We are unable to disable "WISHLIST" using the normal Magento Admin option. Something is coded into Breeze Evolution so that the HEART always appears on the homepage bar. We attempted to remove it using XML but the "HEART" remains and now when you click on it the visitor will see a 404 error.

**Compare**
The same is true of the COMPARE feature. Attempting to remove COMPARE via XML also fails. The compare icon is always present near the ADD TO CART on catalog pages. Worse yet, Breeze Evolution has a spinning wheel while it searches for the products it wants to compare after we've attempted to disable the feature.

**Contact Form**
We were not happy with the change made to the "stock" Magento contact page we were using. We attempted to disable the Breeze page using the "Disable Breeze for specified URLs - One url per line. Example: /helpdesk or ?param=1   but that doesn't work. Now that Magento 2.4+ makes editing the standard contact form more difficult any change that Breeze makes is that much harder to deal with. We can create a new form and call it from a different url (contact-us) but the old version of our contact form was working just fine.

The good news... we are switching from PORTO Theme because of its terrible speed scores... mid 40's with some effort. Even with the small issues we are having with Breeze Evolution the 80+ Page Speed score is worth it. The CLS has dropped from an average of .26 (which Google ranks as failing) to 0.007 which is GREAT!  If SwissUP has "apps" that can be purchased to regain our ability to easily stylize our site we'd love to purchase them.

Aloha

**Comment by vovayatsyuk** [comment]
1. Breeze theme is a theme that doesn't use Blank or Luma styles including email styles. Breeze has its own stylesheets for emails.
2. Thanks, will check it. Meanwhile, you can remove the wishlist from the header with this XML layout update:

    ```xml
    <referenceBlock name="header.wishlist" remove="true"/>
    ```
 
 3. Thanks, will check what's wrong. However, I didn't fully understand this part: "Breeze Evolution has a spinning wheel while it searches for the products it wants to compare".
 4. "Disable Breeze for specified URLs" disables the breeze js part only. Not a theme. May I ask what are you trying to do with the contact page?

**Comment by AlohaJeff** [comment]
1. How can we disable the Breeze Theme style sheets for emails?

2. Fixed. Went back and checked the XML file again and found that it had a typo. Fixed typo and the Wishlist "HEART" is gone.

3. We've used the following in the XML file as we have in the past to remove COMPARE from the site. 
![compare_remove](https://user-images.githubusercontent.com/113044284/193352408-2f904590-2a76-4b85-8819-daa0b45a5475.png)
        
The COMPARE link is still visible on the "home page" next to the Add to Cart button.

4. The sizing of the form fields are too small AND when using Google reCaptcha the formatting is strange. 
![contact_misformat](https://user-images.githubusercontent.com/113044284/193351375-5c94d11e-609b-4c76-a11f-9cd3c37f2177.png)

**New Question:**
When viewing from a mobile device "GRID" products require additional padding. Currently when the product price has 6 or more characters, such as $1234.56 the "6" in the price goes flush up against the product name of the next item. On a desktop even with a screen narrowed down to "mobile" resolution the same thing does not seem to happen.
![more_padding_needed](https://user-images.githubusercontent.com/113044284/193352157-f113e1cc-80bd-4aa2-ade3-f55152c0429f.jpg)

**Reply by vovayatsyuk** [reply]
1. It's not possible.
3. Great! 
4. That doesn't look like our contact form: https://breeze.swissupdemo.com/breeze_evolution/contact#contact-form. You must be using some module that changed styles and made 3 fields in a row. This issue is very simple to solve for a frontend developer.
5. I recommend creating a [custom theme](https://breezefront.com/child-theme) and disabling the product name and price in one row. We have LESS variable for this: https://github.com/breezefront/theme-frontend-breeze-evolution/blob/1.2.0/web/css/theme/abstracts/_variables.less#L168

**Comment by AlohaJeff** [comment]
Thank you for the help. We like "Breeze" enough that we've completed a purchase. We'll most likely use "FORCE" as the Theme as it most closely mimics what we're already displaying to customers. Order # 100017852 via the Swissuplabs website.

Thank you.
